### PR TITLE
chore: update deps and test on node v7

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,20 +19,20 @@
...
-    "autod": "^2.6.1",
+    "autod": "^2.7.1",
     "egg": "*",
-    "egg-bin": "^1.0.2",
-    "egg-ci": "^1.0.2",
-    "egg-mock": "*",
+    "egg-bin": "^1.6.0",
+    "egg-ci": "^1.0.3",
+    "egg-mock": "^1.1.0",
     "egg-security": "*",
```
Comment (popomore) [inline package.json]: 这个不用依赖了？

Comment (fengmk2) [reply package.json]: egg-security 是 fixtures 里面依赖的

Comment (popomore) [reply package.json]: 那把 \* 都改了？

**Comment by fengmk2** [comment]
egg-security 指定依赖版本了

File: app.js [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
- * Module dependencies.
- */
-
 module.exports = app => {
   // put before other core middlewares
   app.config.coreMiddlewares.unshift('cors');
 
   // if security plugin enabled, only allow safe domain support CORS.
-  app.config.cors.origin = function(ctx) {
+  app.config.cors.origin = function corsOrigin(ctx) {
```
Comment (atian25) [inline app.js:8]: arrow function ?

File: package.json [hunk 3]
```diff
@@ -41,12 +39,12 @@
     "test": "npm run lint && npm run test-local",
     "test-local": "egg-bin test",
     "cov": "egg-bin cov",
-    "lint": "eslint --ext js . --fix",
+    "lint": "eslint --ext js .",
```
Comment (atian25) [inline package.json]: `--ext js` 也可以干掉, 是默认值

Comment (atian25) [reply package.json]: `"test": "npm run lint && npm run test-local",` 改为 `"test": "npm run lint -- --fix&& npm run test-local",`

**Comment by atian25** [comment]
修改了下, @fengmk2  review 下

**Comment by fengmk2** [comment]
+1

**Comment by atian25** [comment]
1.0.0

**Pull request merged**